frames/src/main/java/com/checkout/frames/di/component/FramesDIComponent.kt

```diff
@@ -19,7 +22,7 @@ import javax.inject.Named
 import javax.inject.Singleton
 
 @Singleton
-@Component(modules = [ValidationModule::class, StylesModule::class, PaymentModule::class])
+@Component(modules = [ValidationModule::class, StylesModule::class, PaymentModule::class, CardSchemeModule::class])
```

Comment:
Unfortunately, such a change doesn't really make sense. Because CardSchemeModule related objects again in root graph, and we don't have scope separation.
Overall it can be fixed with 2 small changes, in `CardSchemeModule`, `CardSchemeViewModelSubComponent` and `FramesDIComponent`.
Please take a look [Dagger 2, Subcomponent](https://dagger.dev/dev-guide/subcomponents.html)